File: Python Code/Daily_Flash_Code_0/Week 5/13 Feb/27-DailyFlash_Solutions/27-DailyFlash_Solutions/13_Feb_Solution_Three/Java/prog3.java
import java.util.Scanner;

class Demo {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int num = sc.nextInt();
		int temp = num, rem, sum = 0;
		while(temp > 0) {
			rem = temp % 10;
			sum += rem;
			temp /= 10;
		}
		System.out.println("Sum : " + sum);
	}
}

File: Python Code/Daily_Flash_Code_0/Week 5/13 Feb/27-DailyFlash_Solutions/27-DailyFlash_Solutions/13_Feb_Solution_Three/C/prog3.c
#include<stdio.h>

int main(void) {
	int num;
	printf("Enter num : ");
	scanf("%d", &num);
	int temp = num, rem, sum = 0;
	while(temp > 0) {
		rem = temp % 10;
		sum += rem;
		temp /= 10;
	}
	printf("Sum = %d", sum);

	return 0;
}
